aiohttp/helpers.py

```diff
@@ -125,6 +154,12 @@ def proxies_from_env():
             client_logger.warning(
                 "HTTPS proxies %s are not supported, ignoring", proxy)
             continue
+        if netrc_obj and auth is None:
+            auth_from_netrc = netrc_obj.authenticators(proxy.host)
+            if auth_from_netrc is not None:
+                *logins, password = auth_from_netrc
+                auth = BasicAuth(logins[0] if logins[0] else logins[1],
```

Comment:
`logins[-1]` should be enough.
Please add a comment when `logins[0]` can be empty (file is started with tab I guess).

Reply:
https://github.com/requests/requests/blob/24092b11d74af0a766d9cc616622f38adb0044b9/requests/utils.py#L203-L207
`Requests` code shows me how to handle it, and i saw the `netrc` source code.
`netrc.authenticators` method return a (`user`, `account`, `password`) tuple.
I guess `user` and `account` both can be username. 

```python
        if netrc_obj and auth is None:
            auth_from_netrc = netrc_obj.authenticators(proxy.host)
            if auth_from_netrc is not None:
                # auth_from_netrc is a (`user`, `account`, `password`) tuple,
                # `user` and `account` both can be username,
                # if `user` is None, use `account`
                *logins, password = auth_from_netrc
                auth = BasicAuth(logins[0] if logins[0] else logins[-1],
                                 password)
```

Reply:
Ok